assets/js/googlesitekit/modules/datastore/settings.test.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * `googlesitekit/modules` datastore: settings tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Modules from 'googlesitekit-modules';
+import { STORE_NAME } from './constants';
+import { createTestRegistry } from '../../../../../tests/js/utils';
+
+describe( 'core/modules store changes', () => {
+	let registry;
+	const slug = 'test-module';
+	const nonExistentModuleSlug = 'not-module';
+	const moduleStoreName = `modules/${ slug }`;
+	const controlReturn = 'dummy_return_value';
+	const DUMMY_ACTION = 'DUMMY_ACTION';
+	let moduleCanSubmitChanges = false;
+	let submittingChanges = false;
+
+	beforeEach( () => {
+		const storeDefinition = Modules.createModuleStore( moduleStoreName );
+		const canSubmitChanges = () => moduleCanSubmitChanges;
+		const isDoingSubmitChanges = () => {
+			return submittingChanges;
+		};
+
+		registry = createTestRegistry();
+		registry.registerStore( moduleStoreName, Data.combineStores(
+			storeDefinition,
+			{
+				actions: {
+					*submitChanges() {
+						const result = yield {
+							payload: {},
+							type: DUMMY_ACTION,
+						};
+						return result;
+					},
+				},
+				selectors: {
+					canSubmitChanges,
+					isDoingSubmitChanges,
+				},
+				controls: {
+					[ DUMMY_ACTION ]: () => controlReturn,
+				},
+			}
+		) );
+	} );
+
+	describe( 'actions', () => {
+		describe( 'submitChanges', () => {
+			it( 'it proxies the selector call to the module with the given slug', async () => {
```

Comment:
This was probably just an oversight but the `it` shouldn't be part of the string. Other tests have this as well and should be updated.

Also, `submitChanges` is an action so the wording needs to be adapted for accuracy here.
```suggestion
			it( 'proxies the dispatched action to the module with the given slug', async () => {
```